bmu/Src/prechargeDischarge.c

```diff
@@ -653,6 +653,8 @@ void pcdcTask(void *pvParameter)
                 fsmSendEvent(&fsmHandle, EV_Precharge_Finished, portMAX_DELAY);
             } else if (rc == PCDC_ERROR) {
                 DEBUG_PRINT("Precharge Error\n");
+                PrechargeState = 6
```

Comment:
The code doesn't compile, missing semi colon. Try to compile locally before pushing

Reply:
@addison-ch I edited and recompiled the code, sorry about the error.